agent/dockerclient/timeout.go

```diff
@@ -19,6 +19,8 @@ import "time"
 const (
 	// ListContainersTimeout is the timeout for the ListContainers API.
 	ListContainersTimeout = 10 * time.Minute
+	// ListImagesTimeout is the timeout for the ListImages API
+	ListImagesTimeout = 10 * time.Minute
```

Comment:
how did we choose this value? does this hold good for Windows too?

Reply:
will keep this value in this current commit, will test large number of images listImages timeout next.